### PR TITLE
#15 LORM was introduced

### PR DESCRIPTION
**Pull request by skapral**
Fix for #15

### CODE REVIEW

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#109](https://github.com/yegor256/jpeek/issues/109) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#109](https://github.com/yegor256/jpeek/issues/109) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by amihaiemil** [comment]
@skapral looks ok

**Comment by amihaiemil** [comment]
@rultor good to merge

File: src/main/resources/org/jpeek/metrics/LORM.xsl [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            <xsl:variable name="right_ops" select="$right/ops/op[@code='get' or @code='put']"/>
+            <pair>
+              <xsl:value-of select="count($left_ops[.=$right_ops])"/>
+            </pair>
+          </xsl:if>
+        </xsl:for-each>
+      </xsl:for-each>
+    </xsl:variable>
+    <!--
+    @todo #15:30min `conceptual-relations` are pairs of methods in the class
```
Comment (yegor256) [inline src/main/resources/org/jpeek/metrics/LORM.xsl]: @skapral I didn't get it. What is the problem? This text sounds like a documentation, not a description of a problem.

**Comment by yegor256** [comment]
@skapral see my comment above

File: src/main/resources/org/jpeek/metrics/LORM.xsl (hunk 1 above)
Comment (skapral) [reply src/main/resources/org/jpeek/metrics/LORM.xsl]: @yegor256 Updated.

**Comment by skapral** [comment]
@yegor256 applied

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), total is [+90](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8535](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#109](https://github.com/yegor256/jpeek/issues/109) is now out of scope

**Pull request merged**